R/stats.R

```diff
@@ -2,24 +2,21 @@
 #'
 #' @param data A dataframe or matrix. Each row stores the runtime sample points from
 #' several runs
-#' @param max_runtime A Numerical vector. Should have the same size as columns of data
+#' @param max_runtime The budget to use for calculating ERT. If this is a vector, the largest value is taken
```

Comment:
Shall we simply restrict `max_runtime` to an integer?

Reply:
I did it this way to not break any current functionality, but we should indeed deprecate the passing of vectors to this function